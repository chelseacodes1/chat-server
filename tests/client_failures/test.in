REGISTER
LOGIN
CREATE
CREATE not_loggedin 
CREATE "not logged in" 
JOIN
JOIN not_loggedin
SAY
SAY test_channel
SAY test_channel invalid msg