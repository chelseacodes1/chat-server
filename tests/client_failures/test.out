RESULT REGISTER 0
RESULT LOGIN 0
Error: Invalid use of CREATE command
Error: User not logged in, cannot create channel
Error: User not logged in, cannot create channel
Error: Invalid use of JOIN command
RESULT JOIN not_loggedin 0
Error: Invalid use of SAY command
Error: Invalid use of SAY command
Error: user not logged in, cannot send message
